sqs/src/test/scala/akka/stream/alpakka/sqs/scaladsl/SqsSpec.scala

```diff
@@ -188,6 +188,29 @@ class SqsSpec extends FlatSpec with Matchers with DefaultTestContext {
     probe.cancel()
   }
 
+  it should "pull messages from a fifo queue following the same production order" taggedAs Integration in {
+    val queue = randomFifoQueueUrl()
+    implicit val awsSqsClient = sqsClient
+
+    val messages = for (i <- 0 until 10) yield s"Message - $i"
+
+    messages
+      .map(msg => {
+        awsSqsClient
+          .sendMessage(new SendMessageRequest(queue, msg).withMessageGroupId("group1"))
```

Comment:
I used AWS SQS SDK API here because AmazonSQSBufferedAsyncClient currently does not support FIFO Queues. So, I prefer this sync approach instead of mapping over the messages creating a bunch of futures and waiting for those to preserve the initial messages order